isthmus/src/main/java/io/substrait/isthmus/SubstraitRelNodeConverter.java

```diff
@@ -112,6 +113,11 @@ public RelNode visit(NamedScan namedScan) throws RuntimeException {
     return relBuilder.scan(namedScan.getNames()).build();
   }
 
+  @Override
+  public RelNode visit(LocalFiles localFiles) throws RuntimeException {
+    throw new UnsupportedOperationException("LocalFiles is not supported");
```

Comment:
Why are you throwing here?

Reply:
> Why are you throwing here?

Do you  know which Calcite rel to use for `LocalFiles` ?